Makefile

```diff
@@ -19,8 +19,8 @@ all: build run
 dev-registry: check-docker
 	@docker inspect registry >/dev/null 2>&1 && docker start registry || docker run --restart="always" -d -p 5000:5000 --name registry registry:0.9.1
 	@echo
-	@echo "To use local boot2docker registry for Deis development:"
-	@echo "    export DEV_REGISTRY=`boot2docker ip 2>/dev/null`:5000"
+	@echo "To use local Docker Toolbox registry for Deis development:"
```

Comment:
Docker toolbox is the name of the installer, not the components. I'd recommend just `To use local docker registry for Deis development:`